osquery/tables/system/windows/deviceguard_status.cpp

```diff
@@ -57,6 +63,31 @@ QueryData genDeviceGuardStatus(QueryContext& context) {
                                   ? enforcement_methods[umci_status]
                                   : "UNKNOWN";
 
+    std::vector<long> running_security_services;
+    data.GetVectorOfLongs("SecurityServicesRunning", running_security_services);
+    for (int i = 0; i < running_security_services.size(); i++) {
```

Comment:
Why not `boost::algorithm::join`?

Reply:
Would it actually be better in this case? We still have to iterate over the list and do a comparison to convert the integers to string values before adding them to the list, and this roughly matches the style done here previously. Granted I'm not a c++ expert and have little familiarity with boost, so if there's a fancy way to combine the conversion and concatenation I'm open to the idea.

Reply:
I don't know. I guess if we have to the comparison anyhow, there's no win using a `join` :|